controller/env/broker.go

```diff
@@ -69,16 +73,26 @@ func NewBroker(ae *AppEnv, synchronizer RouterSyncStrategy) *Broker {
 	broker.ae.GetStores().ApiSession.GetEventsEmitter().AddListener(db.EventFullyAuthenticated, broker.apiSessionFullyAuthenticated)
 	broker.ae.GetStores().ApiSessionCertificate.AddEntityEventListenerF(broker.apiSessionCertificateCreated, boltz.EntityCreatedAsync)
 	broker.ae.GetStores().ApiSessionCertificate.AddEntityEventListenerF(broker.apiSessionCertificateDeleted, boltz.EntityDeletedAsync)
+	broker.ae.GetStores().Controller.AddEntityEventListenerF(broker.controllerAdded, boltz.EntityCreated)
+	broker.ae.GetStores().Controller.AddEntityEventListenerF(broker.controllerAdded, boltz.EntityUpdated)
 
 	ae.HostController.GetNetwork().AddRouterPresenceHandler(broker)
 	ae.HostController.GetEventDispatcher().AddClusterEventHandler(broker)
 
```

Comment:
Is this doing something?

Reply:
It is meant to replace the pub key updates currently coming from the raft's peer list directly (which may not be a complete list). I have that logic separate from this because it was experimental for quite a bit and one of the last pieces.